Makefile

```diff
@@ -4,6 +4,7 @@ INSTALL_DIR=install -d -m 755
 INSTALL_FILE=install -m 644
 INSTALL_PROGRAM=install -m 755
 INSTALL_SYMLINK=ln -s -f
+SYMLINK_FILE=ln -s
```

Comment:
Because SYMLINK_FILE is used during build target, whereas INSTALL_SYMLINK is used during install target. Both could divert in the future. I'll add comments for both lines.